.github/workflows/vast.yaml

```diff
@@ -152,12 +157,22 @@ jobs:
           if ! is_unchanged cmake/ CMakeLists.txt libvast/ libvast_test/ scripts/ schema/ tools/ vast/ vast.yaml.example version.json; then
             run_vast=true
           fi
-          echo "run-vast=${run_vast}" >> $GITHUB_OUTPUT
           run_vast_plugins=${run_vast}
           if ! is_unchanged plugins/; then
             run_vast_plugins=true
           fi
           echo "run-vast-plugins=${run_vast_plugins}" >> $GITHUB_OUTPUT
+          # Build vast if no artifact is present in google store
+          if [[ ${run_vast_plugins} == "true" && ${run_vast} == "false" ]]; then
+            deb_package="$(echo "vast-${before_version}-$(uname -s)-Release-GCC" | awk '{ print tolower($0) }')"
+            mac_package="$(echo "vast-${before_version}-darwin-Release-Clang" | awk '{ print tolower($0) }')"
```

Comment:
I think it'd actually be better to hardcode it generally since that's much easier to understand. There is little reason to go through `uname -s` here at all. Just makes the job harder to read.